test/check-application

```diff
@@ -942,7 +947,54 @@ class TestFiles(testlib.MachineCase):
         b.click("#paste-into-directory")
         b.mouse("[data-item='newdir']", "dblclick")
         b.wait_visible("[data-item='newfile2']")
-        b.click(".breadcrumb-button:nth-of-type(3)")
+        b.click("[data-item='newfile2']")
+        b.wait_text("#description-list-owner dd", "admin")
+        b.wait_text("#description-list-group dd", "admin")
+
+        # Copy and paste two files
+        b.mouse("[data-item='newfile']", "click", ctrlKey=True)
+        b.mouse("[data-item='newfile']", "contextmenu")
+        b.click(".contextMenu button:contains('Copy')")
+
+        b.click(".breadcrumb-button:nth-of-type(4)")
+        m.execute("runuser -u admin mkdir /home/admin/testdir")
+        b.click("[data-item='testdir']")
+        b.click("#dropdown-menu")
+        b.click("#paste-into-directory")
+
+        b.mouse("[data-item='newdir']", "dblclick")
+        b.wait_visible("[data-item='newfile']")
+        b.wait_visible("[data-item='newfile2']")
```

Comment:
The point of this PR is to retain permissions, so this could check that the testdir/newfile* copies are still owned by admin:admin?